src/flow.h

```diff
@@ -503,6 +503,12 @@ typedef struct Flow_
     uint64_t todstbytecnt;
     uint64_t tosrcbytecnt;
 
+#ifdef HAVE_NDPI
```

Comment:
Could this somehow go into the storage section below ?

Reply:
Perhaps by using `FlowStorageRegister` and friends? Example usage in `app-layer-expectation.c`.